internal/testrunner/runners/system/servicedeployer/terraform.go

```diff
@@ -64,8 +64,14 @@ func addTerraformOutputs(outCtxt ServiceContext) error {
 	// Unmarshall the data into `terraformOutputs`
 	logger.Debug("Unmarshalling terraform output json")
 	var terraformOutputs map[string]OutputMeta
-	if err = json.Unmarshal(content, &terraformOutputs); err != nil {
-		return fmt.Errorf("error during json Unmarshal %w", err)
+
+	if json.Valid(content) {
+		if err = json.Unmarshal(content, &terraformOutputs); err != nil {
+			return fmt.Errorf("error during json Unmarshal %w", err)
+		}
+	} else {
+		logger.Error("Invalid Json Content")
+		return nil
```

Comment:
this could be rewriting to exit earlier.

Would it make sense to just log a Debug message?

```suggestion
    if !json.Valid(content) {
        logger.Error("Invalid JSON Content")
        return nil
    }
	if err = json.Unmarshal(content, &terraformOutputs); err != nil {
		return fmt.Errorf("error during json Unmarshal %w", err)
```